### PR TITLE
Talkative Mothership Interface

### PR DESCRIPTION
**Pull request by mvousden**
See the PR at https://github.com/POETSII/Orchestrator/pull/253.

### CODE REVIEW

File: source/user_guide.md [hunk 1]
```diff
@@ -843,7 +851,8 @@ documentation for a more detailed description of the commands that follow.
 
  - `recall /app`: Given a placed application graph instance (or multiple),
    informs all Motherships that host the application to recall it (forget about
-   it completely), unless it is running (it will need to be stopped first).
+   it completely), unless it is running (it will need to be stopped first), or
+   unless it has been initialised (it will need to be started, then stopped).
```
Comment (heliosfa) [inline source/user_guide.md:855]: We could change this behaviour to allow things at the barrier to respond to KILL packets?

**Review by heliosfa — APPROVED** [review]
Looks like it lines up with the code changes, though I see some RTCL changes snuck in ;)

File: source/user_guide.md (hunk 1 above)
Comment (mvousden) [reply source/user_guide.md:855]: I reckon so

Comment (mvousden) [reply source/user_guide.md:855]: Created #262 so we can discuss further.

**Pull request merged**